src/runtime/mini_mman.h

```diff
@@ -0,0 +1,30 @@
+#ifndef HALIDE_RUNTIME_MMAN_H
```

Comment:
Given that this is a wrapper for mmap/munmap, it seems weird to name it "mman". What's the source of that name?